### PR TITLE
Added bash script to upload bundle to S3

### PR DESCRIPTION
**Pull request by aleksebastian**
Bash script file is ignored from git since it's being ran locally

### CODE REVIEW

File: checkout-service/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+deploy.sh
```
Comment (bshresthap) [inline checkout-service/.gitignore:1]: What is this file for ?

**Review by bshresthap — COMMENTED** [review]
Hi Alek,
Changes look minor. Looks good. Only think I would suggest is not to check in bundle.js file. Any reason you want to check in ?

File: checkout-service/.gitignore (hunk 1 above)
Comment (aleksebastian) [reply checkout-service/.gitignore:1]: It's a bash script file. Like a command you would type in the terminal. I'm using a aws CLI command to upload my updated bundle with an npm command.

Comment (bshresthap) [reply checkout-service/.gitignore:1]: Got it! That's Great!

**Pull request merged**